pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -200,6 +200,11 @@ class RichProgressBar(ProgressBarBase):
     Raises:
         ModuleNotFoundError:
             If required `rich` package is not installed on the device.
+
+    Note:
```

Comment:
noob q: aren't notes supposed to be before `Args` and `Raises`? At least style-wise

Reply:
In our codebase, many instance have it after `Args`.